cmd/restic/cmd_copy.go

```diff
@@ -0,0 +1,230 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/restic/restic/internal/debug"
+	"github.com/restic/restic/internal/errors"
+	"github.com/restic/restic/internal/restic"
+
+	"github.com/spf13/cobra"
+)
+
+var cmdCopy = &cobra.Command{
+	Use:   "copy [flags] [snapshotID ...]",
+	Short: "Copy snapshots from one repository to another",
+	Long: `
+The "copy" command copies one or more snapshots from one repository to another
+repository. Note that this will have to read (download) and write (upload) the
+entire snapshot(s) due to the different encryption keys on the source and
+destination.
```

Comment:
Please add `, and that transferred files are not re-chunked, which may break their deduplication` before the last dot.

Reply:
Is added.